test/spec_lint.rb

```diff
@@ -13,8 +13,12 @@
     [200, { "content-type" => "test/plain", "content-length" => "3" }, ["foo"]]
   end
 
-  def env(*args)
-    Rack::MockRequest.env_for("/", *args)
+  def env(options = {})
+    unless options.key?(:input)
+      options[:input] = String.new
```

Comment:
Lot's of tests fail without it because moist of the tests assume that it's an empty string. If we change that, it will make backporting harder.

Reply:
That should give you a rough indication of how much code this change will break.  If we aren't willing to modify rack's own tests to fully implement the change, it's hard to argue the benefits of the change are worth the cost.

Reply:
> If we aren't willing to modify rack's own tests to fully implement the change

Well, we did modify tack's own tests, and it was just in one place since the test fixture uses this and these tests assume an empty string.